docs/api/cli.rst

```diff
@@ -11,6 +11,6 @@ The commands can be run like follows:
    $ runhouse ssh cluster_name
 
 .. automodule:: runhouse.main
-   :members: login, logout, ssh, notebook, start, restart, status
+   :members: login, logout, ssh, notebook, start, restart, status, cluster_list
```

Comment:
what is this supposed to enable? i dont see `cluster`, `cluster_list` or `list` as a command option when running `runhouse --help`

Reply:
![image](https://github.com/user-attachments/assets/18b76f09-df53-4b5a-a9a0-aa2839b49326)
![image](https://github.com/user-attachments/assets/07472ea3-d3bb-4d0e-8a18-7a359026fce9)